roles/collect/molecule/default/prepare.yml

```diff
@@ -12,9 +16,14 @@
     - name: create test setup.json
       copy:
         dest: '/tmp/ocp-test-config.json'
-        content: '{"ocp_api":"http://localhost:8001", "ocp_user":"test-user", "ocp_token_file":"/path/to/token/file", "cluster_alias": "test-alias", "cluster_id": "test-cluster-id", "ocp_metering_namespace": "metering/test"}'
+        content: '{"ocp_api":"http://localhost:8001", "ocp_user":"test-user", "ocp_token_file":"/tmp/collect-test-token", "cluster_alias": "test-alias", "cluster_id": "test-cluster-id", "ocp_metering_namespace": "metering/test"}'
```

Comment:
After running through test and talking with Andrew and Doug a couple items have been dropped from the config.json. Now this:
```
content: '{"ocp_api":"http://localhost:8001", "ocp_token_file":"/tmp/collect-test-token", "cluster_id": "test-cluster-id", "ocp_metering_namespace": "metering/test"}'
```